### ISSUE TITLE
Incomplete checks for color contrast on text with webkit-line-clamp

### ISSUE THREAD

**Issue by loicplaire** [dequelabs/axe-core#3698]
### Product

axe-core

### Product Version

4.4.2

### Latest Version

- [X] I have tested the issue with the latest version of the product

#### Expectation
No colour contrast violations should be reported when using `webkit-line-clamp` and inner span(s).

#### Actual
A colour contrast incomplete check is reported when using a combination of `webkit-line-clamp` and inner `span`.

#### How to Reproduce

The issue is exactly the same as https://github.com/dequelabs/axe-core/issues/2669 except that this time it happens with the `webkit-line-clamp` CSS property instead of `text-overflow: ellipsis`.

You can reproduce the issue at: https://output.jsbin.com/savapuvoti.

<img width="1426" alt="Screenshot of the Axe dev tools with the violation displayed" src="https://user-images.githubusercontent.com/1925075/193977365-f9a2b1d8-f212-4d62-b364-058f75fc5bc8.png">

**Comment by straker** [comment]
Thanks for the issue.
